.vscode/settings.json

```diff
@@ -4,16 +4,14 @@
     "node_modules/jest-editor-support/src": false,
     "coverage/": true
   },
-  "tslint.exclude": "./integrations/*",
   "search.exclude": {
     "out": true,
     "node_modules/jest-editor-support/src": false
   },
   "typescript.tsdk": "./node_modules/typescript/lib",
-  "eslint.enable": false,
-  "tslint.autoFixOnSave": true,
-  "prettier.semi": false,
-  "prettier.printWidth": 120,
-  "prettier.trailingComma": "es5",
-  "prettier.singleQuote": true
+  // eslint
```

Comment:
Does VSCode complain about the comment in JSON? It's not always allowed. 🤔

Reply:
it's true comments are not part of the official json scheme, but vscode .settings is fine with the comments as long as I can remember...